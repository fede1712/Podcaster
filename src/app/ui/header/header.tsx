import Link from "next/link";

export const Header = () => {
  return (
    <header className="flex justify-between border-b-4 m-10">
      <Link href="/" data-testid="header">
        <h1 className="text-cyan-600 text-xl  font-bold">Podcaster</h1>
      </Link>
    </header>
  );
};
