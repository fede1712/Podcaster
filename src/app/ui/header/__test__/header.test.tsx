import React from "react";
import { render, screen } from "@testing-library/react";
import { Header } from "../header";

describe("Renders header correctly", () => {
  test("should render header correctly", () => {
    render(<Header />);

    const titleElement = screen.queryByText("Podcaster");
    expect(titleElement).not.toBeNull();

    const { asFragment } = render(<Header />);

    expect(asFragment()).toMatchSnapshot();
  });
});
