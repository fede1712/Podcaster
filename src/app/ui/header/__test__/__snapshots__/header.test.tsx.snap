// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders header correctly should render header correctly 1`] = `
<DocumentFragment>
  <header
    class="flex justify-between border-b-4 m-10 "
  >
    <a
      href="/"
    >
      <h1
        class="text-cyan-600 text-xl  font-bold"
      >
        Podcaster
      </h1>
    </a>
  </header>
</DocumentFragment>
`;
